packages/api-explorer/src/index.jsx

```diff
@@ -1,8 +1,19 @@
 import 'antd/dist/antd.css';
 import { Collapse, Tag } from 'antd';
 
+import { IntlProvider, addLocaleData } from 'react-intl';
+import itLocale from 'react-intl/locale-data/it';
+import enLocale from 'react-intl/locale-data/en';
+import it from "../i18n/it.json";
+import en from "../i18n/en.json";
+
 import colors from './colors'
 
+addLocaleData([...itLocale, ...enLocale]);
+const messages = {
```

Comment:
This could be passed as prop to component in order to be able to support any language. If this is internal to ApiExplorer component you limit 
`const localizedMessages` to be only 'it' or 'en'

Reply:
Or you could merge internal 'messages' with an external messages passed as prop

Reply:
Problem found about this note: 
react-intl needs `addLocaleData([...localeEN, ...localeIT])` in order to support a language, so the user should pass xLocale as prop. This does not look like a nice solution. 
Close for now